src/main/webapp/src/app/app.module.ts

```diff
@@ -26,6 +43,11 @@ import {MyGardensComponent} from './my_gardens_page/my-gardens.component';
 import {UserProfileComponent} from './user-profile/user-profile.component';
 import {FindGardensComponent} from './find-gardens-page/find-gardens.component';
 import {SchedulePageComponent} from './schedule-page/schedule-page.component';
+import {LoginComponent} from './login-signup-page/login-signup.component';
+
+const google_oauth_client_id =
```

Comment:
Lets discuss this in chat. We normally don't place secret unique identifiers in source code.

Reply:
Done!